README.md

```diff
@@ -47,6 +47,8 @@ Usernetes aims to provide a reference distribution of Kubernetes that can be ins
 * [ ] Cgroup
 * [X] Multi-node CNI
   * [X] Flannel (VXLAN)
+* [ ] CoreDNS [#73](https://github.com/rootless-containers/usernetes/issues/73)
+* [ ] TLS [#76](https://github.com/rootless-containers/usernetes/issues/76)
```

Comment:
These are already functional in rootless k3s but just not included in the POC scripts in this repo.

Reply:
CoreDNS should be probably mentioned in the paragraph of `56: Deployment shell scripts are in POC status. (It even lacks TLS setup - [#76](https://github.com/rootless-containers/usernetes/issues/76))`